nextflow.config

```diff
@@ -37,6 +43,9 @@ params {
     tpm_filter_dir = '05-CountingFilter/TPM'
     annotation_dir = '06-Annotation'
     deseq2_dir = '07-DifferentialExpression/deseq2'
+    assembly_dir = '08-Assembly/de_novo'
+    rnaseq_annotation_dir = '09-RNA-Seq_Annotation/'
+    full_dir = ''
```

Comment:
also here something more descriptive would be nice. I'm not sure, what would be suitable... `UniRef90_dir`?

Reply:
yeah... I mainly named this `full` because dammit can be run with the default (UniProtKB, fast) or with UniRef90 (slower, more accurate) adding `--full` to the dammit run. But I agree, it's easier to distinguish using a more descriptive parameter. So I also changed this as suggested